consensus/src/consensus/inner/mod.rs

```diff
@@ -72,6 +73,8 @@ impl ConsensusInner {
             return Ok(None);
         }
 
+        //todo: partial ledger fix
```

Comment:
just to check: this is a future TODO, and not something left to do in this PR?

Reply:
already dealt with, will remove